src/ensembl/stories/shared-components/radio/Radio.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import Radio, { RadioOptions } from 'src/shared/components/radio/Radio';
+import { storiesOf } from '@storybook/react';
+import styles from './Radio.stories.scss';
+import { action } from '@storybook/addon-actions';
+
+const radioData: RadioOptions = [
+  { value: 'default', label: 'Default' },
+  { value: 'length_longest', label: 'Spliced length: longest - shortest' },
+  { value: 'length_shortest', label: 'Spliced length: shortest - longest' }
+];
+
+const Wrapper = (props: any) => {
+  const [selectedRadio, setselectedRadio] = useState('default');
+
+  const handleOnchange = (value: string) => {
+    setselectedRadio(value);
+    action('radio-clicked')(value);
+  };
+
+  return (
+    <div>
+      <Radio

```

Comment:
One thing I now realise I missed in the initial commit of this component is its name. I thought the name `Radio` describes a single radiobutton with the label; but now I see it describes the whole group of radio buttons. Does anyone other than me find this confusing? Is it not too late to rename the component to RadioGroup?

Examples in other design systems:
- Ant Design — RadioGroup ([link](https://ant.design/components/radio/#components-radio-demo-radiogroup-options))
- Material UI — RadioGroup ([link](https://material-ui.com/components/radio-buttons/))

Reply:
I thought the same as well but then it clicked in my head that a radio on it own will never exist, they are always a group, so the name sort of mean a group of radio since one of its own will never exist and wont cause any confusion.

Reply:
@jyo @imransl @msmanoj what are your opinions?

Personally, I would strongly prefer RadioGroup.

Reply:
RadioGroup more sense to me than Radio if it is a group of radio buttons.

Reply:
ok going with RadioGroup